sdk/template/ci.yml

```diff
@@ -22,9 +27,11 @@ pr:
 #if (false)
     # The following paths should only be included in template/ci.yml, and removed from any other SDKs which copy this file.
     # The surrounding conditions should accomplish that when installed with `dotnet new azsdk`.
+    # eng/common code changes trigger template pipeline for basic checking.
```

Comment:
Nit: I'd including this comment right before `eng/common`. Makes a little more sense.